nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/registry/flow/mapping/NiFiRegistryFlowMapper.java

```diff
@@ -782,6 +782,7 @@ public VersionedRemoteProcessGroup mapRemoteProcessGroup(final RemoteProcessGrou
         rpg.setProxyHost(remoteGroup.getProxyHost());
         rpg.setProxyPort(remoteGroup.getProxyPort());
         rpg.setProxyUser(remoteGroup.getProxyUser());
+        rpg.setProxyPassword(encrypt(remoteGroup.getProxyPassword()));
```

Comment:
This will always include the proxy password, which we don't want. We only want to include the proxy password if `flowMappingOptions.isMapSensitiveConfiguration()` is true.